ember-flight-icons/addon/styles/app.css

```diff
@@ -1,9 +1,26 @@
-.flight-icon.flight-icon-display-inline {
-  display: inline-block;
-}
 .flight-icon {
   display: block;
 }
+
+.flight-icon-display-inline {
+  display: inline-block;
+}
+
+@keyframes hds-flight-icon-animation-rotation {
+  to {transform: rotate(360deg);}
+}
+
+@media (prefers-reduced-motion: no-preference) {
```

Comment:
[issue, blocker] I don't see a media query that respects `prefers-reduced-motion`